include/sempr/entity/spatial/filter/GeometryFilter.hpp

```diff
@@ -0,0 +1,45 @@
+#ifndef SEMPR_ENTITY_SPATIAL_GEOMETRYFILTER_HPP_
+#define SEMPR_ENTITY_SPATIAL_GEOMETRYFILTER_HPP_
+
+#include <cstddef>
+
+#include <geos/geom/CoordinateSequenceFilter.h>
+#include <geos/geom/CoordinateSequence.h>
+#include <geos/geom/Coordinate.h>
+
+// weeeeeell... it's not an entity. more like... util?
+namespace sempr {
+
+namespace geom = geos::geom;
+
+
+class EnvelopeFilter : public geom::CoordinateSequenceFilter 
```

Comment:
Class <-> File mismatch. `EnvelopeFilter` class in `GeometryFilter.hpp`.
I agree that sometimes the filename should rather match the "theme" of classes declared inside it, but in this case there is only one class... :wink:

Reply:
see GeodeticFilter.